src/figuresClasses.ts

```diff
@@ -1,11 +1,87 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  public shape: string;
+
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    this.shape = 'triangle';
+
+    if (a <= 0 || b <= 0 || c < +0) {
```

Comment:
[CODE STYLE] - The error message here could be more descriptive. It's good practice to provide detailed information about what went wrong. Consider changing the error message to something like 'All sides must have a length greater than 0.'